http/src/request/channel/message/create_message.rs

```diff
@@ -16,9 +16,15 @@ use twilight_model::{
 };
 
 #[derive(Clone, Debug)]
+/// The error created when a messsage can not send.
```

Comment:
I am not sure about the wording here (though english 2nd language LOL) It makes it sound like it was the sending that failed, though this error signals that the message was illegal in the way it would be rejected by Discord.

Reply:
In similar validation error enums I have been writing: `The error created when a channel can not be created as configured`

Reply:
Yeah something like that sounds better to me